mitosheet/css/endo/CellEditor.css

```diff
@@ -41,6 +41,12 @@
     position: absolute;
 }
 
+.jp-RenderedHTMLCommon .mito-container a.cell-editor-dropdown-link {
```

Comment:
Usually we just use a single class name to identify elements to apply css to. I think here you could get away with just using `.cell-editor-dropdown-link`. 

The nice part about just using `.cell-editor-dropdown-link` is that if we use this component in streamlit, for example, the class would still apply even if the `.jp-RenderedHTMLCommon` isn't there. 

Are there advantages to using this more specific class selection technique that aren't on our radar right now?

Reply:
Ah yeah that was because it was being overridden by the JupyterLab stuff (I usually try to make it as brief as possible too!). I saw this a little in other parts of the CSS so I think it might be something you've run into before: https://github.com/mito-ds/monorepo/blob/de5decfdd6c935c33705f53f21713de53e983e7b/mitosheet/css/mito.css#L20